app/views/species_lists/bulk_editor.html.erb

```diff
@@ -128,7 +128,7 @@
   <div id="row_controls" class="row_controls">
     <%= :OBSERVATION.t %>:
     <%=
-      args = Observation.show_link_args("XXX")
+      args = observation_path("XXX")
```

Comment:
I think this is an accurate "translation" of the original.
But I have no idea how the original worked or whether the new version will work.

Reply:
Yes this is odd. It may be used by a script... I'll check it out now.

Reply:
Yes. JS checks for the XXX and replaces it with current_id (selected by the user)

Reply:
Thanks for checking this out!